packages/sdk/README.md

```diff
@@ -300,6 +314,20 @@ const key = 'my-key'
 const data = await sdk.storage.get(key)
 ```
 
+### storage.delete()
+
+Deletes a file from the app's manifest.
```

Comment:
Could you use another term than "manifest" here please? It's an implementation detail, I don't think it should leak to the doc.

Reply:
I actually paused when I was writing that, but wasn't sure what would be better.

Maybe "Deletes an entry from the list of files available to the app"?